### PR TITLE
Allow display of Fahrenheit on dashboard

### PR DESCRIPTION
**Pull request by miken32**
- [x] Redmine Issue: https://redmine.pfsense.org/issues/8205
- [x] Ready for review

### CODE REVIEW

**Comment by sbeaver-netgate** [comment]
A number of people have asked for this so thanks for the work. I wish we could drag the USA (and Albania) out of the Victorian era, but the principle unit of measurement here still being the "football field", that is a losing battle :)

I found that when clicking the "Show temperature in Fahrenheit" checkbox the temperatures all convert via JQuery, but the text label still says: "Thresholds in °C (1 to 100):"  Having gone to the trouble of doing that JQuery, it seems a shame not to fix that.

**Comment by sbeaver-netgate** [comment]
When F is selected, the color coding is no longer applied to the progress bars.
pfsense_winemack_net_-_status__dashboard

**Comment by miken32** [comment]
@sbeaver-netgate , threshold colours are tested working now; there was a difference between my test router and what I'd uploaded here. Label is updated by the checkbox now (with correct internationalization) and I addressed your comments re the polling object (I had no idea what those options were for, but I'll assume you do!)

**Comment by sbeaver-netgate** [comment]
Thanks. "name' just gives this timer instance a label which is helpful in debugging. "frequency" (which I realize now I should have called "period") specifies how often the AJAX call is made. Bigger the number, the less often it is made.

File: src/usr/local/www/widgets/widgets/thermal_sensors.widget.php [hunk 1]
```diff
@@ -203,7 +240,9 @@ function ts_callback(s) {
 		<input type="hidden" name="widgetkey" value="<?=htmlspecialchars($widgetkey); ?>">
 		<?=gen_customwidgettitle_div($widgetconfig['title']); ?>
 		<div class="form-group">
-			<label class="col-sm-6 control-label"><?=gettext('Thresholds in')?> &deg;C <?=gettext('(1 to 100):')?></label>
+			<label class="col-sm-6 control-label">
+				<?=sprintf(gettext('Thresholds in <span class="thermal_sensors_widget_unit">%s</span><br/><span class="thermal_sensors_widget_range">%s</span>:'), $thermal_sensors_widget_showFahrenheit ? gettext("&deg;F") : gettext("&deg;C"), $thermal_sensors_widget_showFahrenheit ? gettext("(1&ndash;212)") : gettext("(1&ndash;100)"))?>
+			</label>
```
Comment (sbeaver-netgate) [inline src/usr/local/www/widgets/widgets/thermal_sensors.widget.php]: I know I am being picky, but replacing the <br /> here with a, &nbsp; looks a little nicer I think. No problem if you disagree. I will merge this shortly.

Comment (miken32) [reply src/usr/local/www/widgets/widgets/thermal_sensors.widget.php]: Github ate your comment!

Comment (sbeaver-netgate) [reply src/usr/local/www/widgets/widgets/thermal_sensors.widget.php]: Hah! I substitutes a non-breaking space there :) & n b s p ;

**Pull request merged**